test/controllers/api/jobs_controller_test.rb

```diff
@@ -87,6 +87,11 @@ def doorkeeper_token=(t)
     response.must_be :success?
   end
 
+  test "should list jobs by page and application" do
+    get :index, format: :json, page: 1, status: 'incomplete'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.